pkgs/development/python-modules/diagrams/default.nix

```
@@ -0,0 +1,66 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+#, poetry
+, bash
+, black
+, jinja2
+, round
+, graphviz
+, inkscape
+, imagemagick
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "diagrams";
+  version = "0.17.0";
+  format = "pyproject";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "mingrammer";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1h5i9xb4fsmy81sznvqycxl712d9k695vznkpx2wm1ki7hvz86s8";
+  };
+
+  preConfigure = ''
+    sed -i "s@#!/bin/bash@$!/${bash}/bin/bash@" autogen.sh
+    ./autogen.sh
+  '';
```

```suggestion
  preConfigure = ''
    patchShebang autogen.sh
    ./autogen.sh
  '';
```
this should alsow work